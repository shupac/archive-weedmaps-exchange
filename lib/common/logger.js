const Logger = require('js-logger');
const serializeError = require('serialize-error');

const env = process.env.NODE_ENV || 'development';
const defaultLevel = process.env.LOG_LEVEL
  ? Logger[process.env.LOG_LEVEL.toUpperCase()]
  : Logger.DEBUG;

// Set the log level to what the ENV is set to or debug
Logger.useDefaults({
  defaultLevel,
  formatter: (messages, context) => {
    let args = false;
    if (messages.length > 1) {
      args = messages.splice(1);
    }
    const log = {
      message: messages[0],
      component: context.name
        ? context.name.replace('@ghostgroup/admin:', 'APP:')
        : '-',
      data: args === false ? undefined : args,
      log_level: context.level.name,
      timestamp: new Date().toUTCString(),
    };
    if (env !== 'development') {
      try {
        messages[0] = JSON.stringify(log);
      } catch (e) {
        log.message = serializeError(log.message);
        log.data = serializeError(log.data);
        messages[0] = JSON.stringify(log);
      }
    } else {
      messages[0] = `[${log.timestamp}] <${log.log_level}> (${
        log.component
      }): `;
      if (typeof log.message === 'string') {
        messages[0] = `${messages[0]} ${log.message}`;
        if (args) {
          messages[1] = log.data;
        }
      } else {
        messages[1] = log.message;
        if (args) {
          messages[2] = log.data;
        }
      }
    }
  },
});

module.exports = Logger;
