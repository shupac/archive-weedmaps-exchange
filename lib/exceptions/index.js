export { UnauthenticatedUserException } from './auth';
export { NotFoundException } from './not-found';
