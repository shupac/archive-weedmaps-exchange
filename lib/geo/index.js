// @flow
import geojsonExtent from '@mapbox/geojson-extent';

export type Point = [number, number];
export type Geometry = {
  type: string,
  coordinates: Point[][] | Point[][][],
};

export type Bounds = {
  ne: {
    lat: number,
    lng: number,
  },
  sw: {
    lat: number,
    lng: number,
  },
};

export type WSENBounds = [number, number, number, number];
export const INITIAL_DEFAULT_MAP_ZOOM_LEVEL = 10;
export const MAX_ZOOM = 18;
export const DEFAULT_CENTER = [-118.3583923, 34.0225723];

/**
 * Get a bounding box from a set of Geometries
 * @param data
 */
export function getBoundingBox(geometries: Geometry[]): WSENBounds {
  return geojsonExtent({ type: 'GeometryCollection', geometries });
}
