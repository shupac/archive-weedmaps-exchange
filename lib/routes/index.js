const nextRoutes = require('@ghostgroup/next-routes');

const routes = nextRoutes();

routes.add('home', '/', 'index');
routes.add('help', '/help', 'help');

// BUYER
routes.add(
  'marketplace',
  '/buyer/marketplace/:tab(discover|catalog|watchlist)?',
  'marketplace',
);
routes.add(
  'marketplaceProduct',
  '/buyer/marketplace/:tab(catalog)/product/:productId',
  'marketplace',
);

routes.add('cart', '/buyer/cart', 'cart');

routes.add('cartConfirmation', '/buyer/cart/confirmation/:orderId', 'cart');

routes.add('buyerOrders', '/buyer/orders', 'buyer-orders');

routes.add('buyerOrder', '/buyer/orders/:orderId', 'buyer-orders');

routes.add('settings', '/buyer/settings/:tab(profile|locations)?', 'settings');

// SELLER
routes.add('sellerOrders', '/seller/orders', 'seller-orders');

routes.add('sellerOrder', '/seller/orders/:orderId', 'seller-orders');

routes.add('sellerProducts', '/seller/products', 'seller-products');

routes.add('sellerProduct', '/seller/products/:productId', 'seller-products');

routes.add(
  'sellerSettings',
  '/seller/settings/:tab(general|profile|zones)?',
  'seller-settings',
);

routes.config = {};

module.exports = routes;
