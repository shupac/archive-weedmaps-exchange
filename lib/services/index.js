// export { default as Auth } from 'lib/services/auth';
