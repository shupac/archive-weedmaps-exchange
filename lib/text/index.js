// abstract out our content to support future localization
const Content = {
  fourOhFourSubHeader: 'Page does not exist',
  fourOhFourContent1: 'Sorry, the page you are looking for does not exist,',
  fourOhFourContent2: 'but feel free to explore more below',
  newsletterLabel: 'Sign up for newsletter',
  newsletterInputPlaceholder: 'Enter Email Address',
};

export default Content;
