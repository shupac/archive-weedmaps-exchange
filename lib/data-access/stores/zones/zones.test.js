import * as MST from 'mobx-state-tree';
import * as Geo from 'lib/geo';
import logger from 'lib/common/logger';
import mockZones from 'lib/mocks/zones';
import mockRegions from 'lib/mocks/regions';
import mockRegion from 'lib/mocks/region';
import Zones from './';

describe('Zones store', () => {
  afterEach(() => {
    jest.clearAllMocks();
  });

  describe('mutate zones', () => {
    it('can fetch zones data', async done => {
      const mockClient = {
        fetch: jest.fn().mockResolvedValue({ data: mockZones }),
      };
      const zones = Zones.create({}, { client: mockClient });
      await zones.fetchZones();
      expect(mockClient.fetch).toHaveBeenCalledWith('/seller/zones');
      expect(zones.zones.length).toBe(mockZones.length);
      done();
    });

    it('can add a zone', () => {
      const zones = Zones.create({});
      zones.addZone(mockZones[0]);
      expect(zones.zones.length).toEqual(1);
    });

    it('can remove a zone', () => {
      const zones = Zones.create({ zones: mockZones });
      zones.removeZone(zones.zones[0]);
      expect(zones.zones.length).toEqual(mockZones.length - 1);
    });

    it('can delete a zone', async done => {
      const zones = Zones.create({ zones: mockZones });
      const zone = zones.zones[0];
      const zoneDelete = jest.spyOn(zone, 'delete').mockResolvedValue();
      const destroy = jest.spyOn(MST, 'destroy');
      await zones.deleteZone(zone);
      expect(zoneDelete).toHaveBeenCalled();
      expect(destroy).toHaveBeenCalledWith(zone);
      done();
    });

    it('can handle errors when fetching zones', async done => {
      const error = new Error('Test');
      const mockClient = {
        fetch: jest.fn().mockRejectedValue(error),
      };
      const zones = Zones.create({}, { client: mockClient });
      const logError = jest.spyOn(logger, 'error').mockReturnValue();
      await zones.fetchZones();
      expect(mockClient.fetch).toHaveBeenCalledWith('/seller/zones');
      expect(logError).toHaveBeenCalledWith(error);
      expect(zones.zones.length).toBe(0);
      done();
    });

    it('can handle errors when deleting a zone', async done => {
      const zones = Zones.create({ zones: mockZones });
      const zone = zones.zones[0];
      const error = new Error('Test');
      const zoneDelete = jest.spyOn(zone, 'delete').mockRejectedValue(error);
      const destroy = jest.spyOn(MST, 'destroy');
      const logError = jest.spyOn(logger, 'error').mockReturnValue();
      await zones.deleteZone(zone);
      expect(zoneDelete).toHaveBeenCalled();
      expect(destroy).not.toHaveBeenCalled();
      expect(logError).toHaveBeenCalledWith(error);
      done();
    });
  });

  describe('fetch regions', () => {
    it('can fetch regions for zones', async done => {
      const mockSdk = {
        // $FlowFixMe
        fetch: jest.fn().mockResolvedValue({
          json: () => Promise.resolve({ data: mockRegion }),
        }),
      };
      const zones = Zones.create({ zones: mockZones }, { wmSdk: mockSdk });
      await zones.fetchRegionsForZones();
      expect(MST.getSnapshot(zones.regionsWithGeometry)).toMatchSnapshot();
      done();
    });

    it('can fetch region geometries with custom bounding box', async done => {
      const mockSdk = {
        // $FlowFixMe
        fetch: async url => {
          const mockData = url.includes('/regions?') ? mockRegions : mockRegion;

          return {
            json: () => Promise.resolve({ data: mockData }),
          };
        },
      };
      const zones = Zones.create({ zones: mockZones }, { wmSdk: mockSdk });
      await zones.fetchRegionsForZones();
      const customBbox = [33.71, -118.58, 34.33, -118.13];
      await zones.fetchRegionsInBounds(customBbox);
      expect(MST.getSnapshot(zones.regionsWithGeometry)).toMatchSnapshot();
      done();
    });

    it('can fetch region geometries without bounding box', async done => {
      const mockSdk = {
        // $FlowFixMe
        fetch: async url => {
          const mockData = url.includes('/regions?') ? mockRegions : mockRegion;

          return {
            json: () => Promise.resolve({ data: mockData }),
          };
        },
      };
      const zones = Zones.create({ zones: mockZones }, { wmSdk: mockSdk });
      await zones.fetchRegionsForZones();
      await zones.fetchRegionsInBounds();
      expect(MST.getSnapshot(zones.regionsWithGeometry)).toMatchSnapshot();
      done();
    });

    it('can fetch region geometries with empty bounding box', async done => {
      jest.spyOn(Geo, 'getBoundingBox').mockReturnValue();
      const mockSdk = {
        // $FlowFixMe
        fetch: async url => {
          const mockData = url.includes('/regions?') ? mockRegions : mockRegion;

          return {
            json: () => Promise.resolve({ data: mockData }),
          };
        },
      };
      const zones = Zones.create({ zones: mockZones }, { wmSdk: mockSdk });
      await zones.fetchRegionsForZones();
      await zones.fetchRegionsInBounds();
      expect(MST.getSnapshot(zones.regionsWithGeometry)).toMatchSnapshot();
      done();
    });

    it('can fetch region geometries for empty regions', async done => {
      const mockSdk = {
        // $FlowFixMe
        fetch: async url => {
          const mockData = url.includes('/regions?') ? mockRegions : mockRegion;

          return {
            json: () => Promise.resolve({ data: mockData }),
          };
        },
      };
      const zones = Zones.create({}, { wmSdk: mockSdk });
      await zones.fetchRegionsForZones();
      await zones.fetchRegionsInBounds();
      expect(MST.getSnapshot(zones.regionsWithGeometry)).toMatchSnapshot();
      done();
    });
  });

  describe('views', () => {
    let zones;

    beforeEach(async () => {
      const mockSdk = {
        // $FlowFixMe
        fetch: async url => {
          const mockData = url.includes('/regions?') ? mockRegions : mockRegion;

          return {
            json: () => Promise.resolve({ data: mockData }),
          };
        },
      };
      zones = Zones.create({ zones: mockZones }, { wmSdk: mockSdk });
      await zones.fetchRegionsForZones();
    });

    it('can compute regionsWithoutZones', () => {
      expect(zones.regionsWithoutZones).toMatchSnapshot();
    });

    it('can compute zoneRegions', () => {
      expect(zones.zoneRegions).toMatchSnapshot();
    });

    it('can compute zoneRegionsById', () => {
      expect(zones.zoneRegionsById).toMatchSnapshot();
    });
  });
});
