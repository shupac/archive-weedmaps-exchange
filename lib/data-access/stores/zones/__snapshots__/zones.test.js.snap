// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Zones store fetch regions can fetch region geometries for empty regions 1`] = `
Object {
  "3": Object {
    "geometry": Object {
      "coordinates": Array [],
      "type": "MultiPolygon",
    },
    "id": 3,
    "name": "5 Cities",
  },
  "4": Object {
    "geometry": Object {
      "coordinates": Array [],
      "type": "MultiPolygon",
    },
    "id": 4,
    "name": "Arizona",
  },
  "5": Object {
    "geometry": Object {
      "coordinates": Array [],
      "type": "Polygon",
    },
    "id": 5,
    "name": "Beach Cities Brands",
  },
  "6": Object {
    "geometry": Object {
      "coordinates": Array [],
      "type": "Polygon",
    },
    "id": 6,
    "name": "Central California Brands",
  },
  "7": Object {
    "geometry": Object {
      "coordinates": Array [],
      "type": "MultiPolygon",
    },
    "id": 7,
    "name": "Downtown LA Brands",
  },
}
`;

exports[`Zones store fetch regions can fetch region geometries with custom bounding box 1`] = `
Object {
  "3": Object {
    "geometry": Object {
      "coordinates": Array [],
      "type": "MultiPolygon",
    },
    "id": 3,
    "name": "5 Cities",
  },
  "4": Object {
    "geometry": Object {
      "coordinates": Array [],
      "type": "MultiPolygon",
    },
    "id": 4,
    "name": "Arizona",
  },
  "5": Object {
    "geometry": Object {
      "coordinates": Array [],
      "type": "Polygon",
    },
    "id": 5,
    "name": "Beach Cities Brands",
  },
  "6": Object {
    "geometry": Object {
      "coordinates": Array [],
      "type": "Polygon",
    },
    "id": 6,
    "name": "Central California Brands",
  },
  "7": Object {
    "geometry": Object {
      "coordinates": Array [],
      "type": "MultiPolygon",
    },
    "id": 7,
    "name": "Downtown LA Brands",
  },
}
`;

exports[`Zones store fetch regions can fetch region geometries with empty bounding box 1`] = `
Object {
  "3": Object {
    "geometry": Object {
      "coordinates": Array [],
      "type": "MultiPolygon",
    },
    "id": 3,
    "name": "5 Cities",
  },
  "4": Object {
    "geometry": Object {
      "coordinates": Array [],
      "type": "MultiPolygon",
    },
    "id": 4,
    "name": "Arizona",
  },
  "5": Object {
    "geometry": Object {
      "coordinates": Array [],
      "type": "Polygon",
    },
    "id": 5,
    "name": "Beach Cities Brands",
  },
  "6": Object {
    "geometry": Object {
      "coordinates": Array [],
      "type": "Polygon",
    },
    "id": 6,
    "name": "Central California Brands",
  },
  "7": Object {
    "geometry": Object {
      "coordinates": Array [],
      "type": "MultiPolygon",
    },
    "id": 7,
    "name": "Downtown LA Brands",
  },
}
`;

exports[`Zones store fetch regions can fetch region geometries without bounding box 1`] = `
Object {
  "3": Object {
    "geometry": Object {
      "coordinates": Array [],
      "type": "MultiPolygon",
    },
    "id": 3,
    "name": "5 Cities",
  },
  "4": Object {
    "geometry": Object {
      "coordinates": Array [],
      "type": "MultiPolygon",
    },
    "id": 4,
    "name": "Arizona",
  },
  "5": Object {
    "geometry": Object {
      "coordinates": Array [],
      "type": "Polygon",
    },
    "id": 5,
    "name": "Beach Cities Brands",
  },
  "6": Object {
    "geometry": Object {
      "coordinates": Array [],
      "type": "Polygon",
    },
    "id": 6,
    "name": "Central California Brands",
  },
  "7": Object {
    "geometry": Object {
      "coordinates": Array [],
      "type": "MultiPolygon",
    },
    "id": 7,
    "name": "Downtown LA Brands",
  },
}
`;

exports[`Zones store fetch regions can fetch regions for zones 1`] = `
Object {
  "3": Object {
    "geometry": Object {
      "coordinates": Array [
        Array [
          Array [
            -117.75257,
            33.59035,
          ],
          Array [
            -117.75297,
            33.59007,
          ],
          Array [
            -117.75338,
            33.58962,
          ],
          Array [
            -117.754191,
            33.588351,
          ],
          Array [
            -117.753387,
            33.58794,
          ],
          Array [
            -117.750067,
            33.586206,
          ],
          Array [
            -117.750061,
            33.583988,
          ],
          Array [
            -117.749015,
            33.583556,
          ],
          Array [
            -117.749458,
            33.583346,
          ],
          Array [
            -117.749953,
            33.582821,
          ],
        ],
      ],
      "type": "Polygon",
    },
    "id": 3,
    "name": "South OC",
  },
  "4": Object {
    "geometry": Object {
      "coordinates": Array [
        Array [
          Array [
            -117.75257,
            33.59035,
          ],
          Array [
            -117.75297,
            33.59007,
          ],
          Array [
            -117.75338,
            33.58962,
          ],
          Array [
            -117.754191,
            33.588351,
          ],
          Array [
            -117.753387,
            33.58794,
          ],
          Array [
            -117.750067,
            33.586206,
          ],
          Array [
            -117.750061,
            33.583988,
          ],
          Array [
            -117.749015,
            33.583556,
          ],
          Array [
            -117.749458,
            33.583346,
          ],
          Array [
            -117.749953,
            33.582821,
          ],
        ],
      ],
      "type": "Polygon",
    },
    "id": 4,
    "name": "Central California",
  },
  "5": Object {
    "geometry": Object {
      "coordinates": Array [
        Array [
          Array [
            -117.75257,
            33.59035,
          ],
          Array [
            -117.75297,
            33.59007,
          ],
          Array [
            -117.75338,
            33.58962,
          ],
          Array [
            -117.754191,
            33.588351,
          ],
          Array [
            -117.753387,
            33.58794,
          ],
          Array [
            -117.750067,
            33.586206,
          ],
          Array [
            -117.750061,
            33.583988,
          ],
          Array [
            -117.749015,
            33.583556,
          ],
          Array [
            -117.749458,
            33.583346,
          ],
          Array [
            -117.749953,
            33.582821,
          ],
        ],
      ],
      "type": "Polygon",
    },
    "id": 5,
    "name": "Oregon",
  },
  "6": Object {
    "geometry": Object {
      "coordinates": Array [
        Array [
          Array [
            -117.75257,
            33.59035,
          ],
          Array [
            -117.75297,
            33.59007,
          ],
          Array [
            -117.75338,
            33.58962,
          ],
          Array [
            -117.754191,
            33.588351,
          ],
          Array [
            -117.753387,
            33.58794,
          ],
          Array [
            -117.750067,
            33.586206,
          ],
          Array [
            -117.750061,
            33.583988,
          ],
          Array [
            -117.749015,
            33.583556,
          ],
          Array [
            -117.749458,
            33.583346,
          ],
          Array [
            -117.749953,
            33.582821,
          ],
        ],
      ],
      "type": "Polygon",
    },
    "id": 6,
    "name": "Arkansas",
  },
  "7": Object {
    "geometry": Object {
      "coordinates": Array [
        Array [
          Array [
            -117.75257,
            33.59035,
          ],
          Array [
            -117.75297,
            33.59007,
          ],
          Array [
            -117.75338,
            33.58962,
          ],
          Array [
            -117.754191,
            33.588351,
          ],
          Array [
            -117.753387,
            33.58794,
          ],
          Array [
            -117.750067,
            33.586206,
          ],
          Array [
            -117.750061,
            33.583988,
          ],
          Array [
            -117.749015,
            33.583556,
          ],
          Array [
            -117.749458,
            33.583346,
          ],
          Array [
            -117.749953,
            33.582821,
          ],
        ],
      ],
      "type": "Polygon",
    },
    "id": 7,
    "name": "Texas",
  },
}
`;

exports[`Zones store views can compute regionsWithoutZones 1`] = `Array []`;

exports[`Zones store views can compute zoneRegions 1`] = `
Array [
  Object {
    "id": "64d05017-4339-4cda-9e57-0da061bf6b00",
    "name": "South OC",
    "wmRegionId": 3,
  },
  Object {
    "id": "2411d92b-d816-4d63-be75-d1a526cb84b3",
    "name": "Central California",
    "wmRegionId": 4,
  },
  Object {
    "id": "f72ef8f0-3ba0-40d7-b328-613ab851bc26",
    "name": "Oregon",
    "wmRegionId": 5,
  },
  Object {
    "id": "fd167cb8-5be3-4c5e-9207-1e78b771c11d",
    "name": "Arkansas",
    "wmRegionId": 6,
  },
  Object {
    "id": "edd2ef7e-00b4-444d-b3e3-996ccbaa7b95",
    "name": "Texas",
    "wmRegionId": 7,
  },
]
`;

exports[`Zones store views can compute zoneRegionsById 1`] = `
Object {
  "3": Object {
    "region": Object {
      "id": "64d05017-4339-4cda-9e57-0da061bf6b00",
      "name": "South OC",
      "wmRegionId": 3,
    },
    "zone": Object {
      "cId": "78d2a139-bf28-4570-a787-be3d3b297487",
      "color": "#F5A623",
      "id": "78d2a139-bf28-4570-a787-be3d3b297487",
      "name": "SoCal",
      "regions": Array [
        Object {
          "id": "64d05017-4339-4cda-9e57-0da061bf6b00",
          "name": "South OC",
          "wmRegionId": 3,
        },
      ],
    },
  },
  "4": Object {
    "region": Object {
      "id": "2411d92b-d816-4d63-be75-d1a526cb84b3",
      "name": "Central California",
      "wmRegionId": 4,
    },
    "zone": Object {
      "cId": "89cfc0ca-4410-49e2-af3a-f74808926bbd",
      "color": "#D0021B",
      "id": "89cfc0ca-4410-49e2-af3a-f74808926bbd",
      "name": "Andrew's Zone #3 3449 Updated 5",
      "regions": Array [
        Object {
          "id": "2411d92b-d816-4d63-be75-d1a526cb84b3",
          "name": "Central California",
          "wmRegionId": 4,
        },
        Object {
          "id": "f72ef8f0-3ba0-40d7-b328-613ab851bc26",
          "name": "Oregon",
          "wmRegionId": 5,
        },
        Object {
          "id": "fd167cb8-5be3-4c5e-9207-1e78b771c11d",
          "name": "Arkansas",
          "wmRegionId": 6,
        },
        Object {
          "id": "edd2ef7e-00b4-444d-b3e3-996ccbaa7b95",
          "name": "Texas",
          "wmRegionId": 7,
        },
      ],
    },
  },
  "5": Object {
    "region": Object {
      "id": "f72ef8f0-3ba0-40d7-b328-613ab851bc26",
      "name": "Oregon",
      "wmRegionId": 5,
    },
    "zone": Object {
      "cId": "89cfc0ca-4410-49e2-af3a-f74808926bbd",
      "color": "#D0021B",
      "id": "89cfc0ca-4410-49e2-af3a-f74808926bbd",
      "name": "Andrew's Zone #3 3449 Updated 5",
      "regions": Array [
        Object {
          "id": "2411d92b-d816-4d63-be75-d1a526cb84b3",
          "name": "Central California",
          "wmRegionId": 4,
        },
        Object {
          "id": "f72ef8f0-3ba0-40d7-b328-613ab851bc26",
          "name": "Oregon",
          "wmRegionId": 5,
        },
        Object {
          "id": "fd167cb8-5be3-4c5e-9207-1e78b771c11d",
          "name": "Arkansas",
          "wmRegionId": 6,
        },
        Object {
          "id": "edd2ef7e-00b4-444d-b3e3-996ccbaa7b95",
          "name": "Texas",
          "wmRegionId": 7,
        },
      ],
    },
  },
  "6": Object {
    "region": Object {
      "id": "fd167cb8-5be3-4c5e-9207-1e78b771c11d",
      "name": "Arkansas",
      "wmRegionId": 6,
    },
    "zone": Object {
      "cId": "89cfc0ca-4410-49e2-af3a-f74808926bbd",
      "color": "#D0021B",
      "id": "89cfc0ca-4410-49e2-af3a-f74808926bbd",
      "name": "Andrew's Zone #3 3449 Updated 5",
      "regions": Array [
        Object {
          "id": "2411d92b-d816-4d63-be75-d1a526cb84b3",
          "name": "Central California",
          "wmRegionId": 4,
        },
        Object {
          "id": "f72ef8f0-3ba0-40d7-b328-613ab851bc26",
          "name": "Oregon",
          "wmRegionId": 5,
        },
        Object {
          "id": "fd167cb8-5be3-4c5e-9207-1e78b771c11d",
          "name": "Arkansas",
          "wmRegionId": 6,
        },
        Object {
          "id": "edd2ef7e-00b4-444d-b3e3-996ccbaa7b95",
          "name": "Texas",
          "wmRegionId": 7,
        },
      ],
    },
  },
  "7": Object {
    "region": Object {
      "id": "edd2ef7e-00b4-444d-b3e3-996ccbaa7b95",
      "name": "Texas",
      "wmRegionId": 7,
    },
    "zone": Object {
      "cId": "89cfc0ca-4410-49e2-af3a-f74808926bbd",
      "color": "#D0021B",
      "id": "89cfc0ca-4410-49e2-af3a-f74808926bbd",
      "name": "Andrew's Zone #3 3449 Updated 5",
      "regions": Array [
        Object {
          "id": "2411d92b-d816-4d63-be75-d1a526cb84b3",
          "name": "Central California",
          "wmRegionId": 4,
        },
        Object {
          "id": "f72ef8f0-3ba0-40d7-b328-613ab851bc26",
          "name": "Oregon",
          "wmRegionId": 5,
        },
        Object {
          "id": "fd167cb8-5be3-4c5e-9207-1e78b771c11d",
          "name": "Arkansas",
          "wmRegionId": 6,
        },
        Object {
          "id": "edd2ef7e-00b4-444d-b3e3-996ccbaa7b95",
          "name": "Texas",
          "wmRegionId": 7,
        },
      ],
    },
  },
}
`;
